src/Microsoft.Azure.SignalRService/Scope.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalRService
+{
+    public enum Scope
+    {
+        Client, /// scope for  client to connect to a hub
+        Server, /// scope for all server side operations
+
+        /// compatible with current runtime's scope
+        Broadcast,
```

Comment:
>Broadcast [](start = 8, length = 9)

should this be another dimension, like ActionScope?

Reply:
`Sever` scope cover all operations on a hub.

`Broadcast`, ... , `RemoveConnectionFromGroup` are also scopes for server side operation. 

But most of them are much smaller scopes than `Server` scope.

E.g. `SendToUser` only cover the scope of sending to specific user.

These small scopes is used to be compatible to current signalr service. In the future, scope system will be redesigned, which at least contains two scopes `Server` and `Client`. 

How about adding prefix 'Server' to these small scopes, since they are specific operations on server side. e.g. `ServerSendToUser`.

Reply:
Remove `Scope` since we consider the service provides the connection string with scope in the future. [Details here](https://github.com/Azure/azure-signalr/pull/289#discussion_r243140488)